dev-docs/explanations/string_changes_policy.md

```diff
@@ -0,0 +1,30 @@
+# Policy on changing user-facing strings in Ubuntu Pro Client
+
+As of version 30 of `pro`, we support translations of user-facing strings throughout the program.
+These are maintained in `po` files in the `debian/po` directory of the git repository.
+Whenever a string is changed, added, or deleted, that has an impact on existing translations and
+therefore an impact on the user experience of `pro` in non-english locales.
+To minimize regressions in translations, we have the following policy in place regarding string changes.
+This policy is implemented in practice via a custom Github action that posts a questionare and prompt
+for justification on all Github PRs that modify the `messages` module of Ubuntu Pro Client.
```

Comment:
```suggestion
for justification on all GitHub PRs that modify the `messages` module of Ubuntu Pro Client.
```

Reply:
I'm just gonna suggest a global search/replace for this :)

Reply:
Done :)